yt_dlp/extractor/generic.py

```diff
@@ -924,28 +924,19 @@ class GenericIE(InfoExtractor):
                 'title': 'EP3S5 - Bon Appétit - Baqueira Mi Corazon !',
             }
         },
-        # Kaltura embed (different embed code)
-        {
-            'url': 'http://www.premierchristianradio.com/Shows/Saturday/Unbelievable/Conference-Videos/Os-Guinness-Is-It-Fools-Talk-Unbelievable-Conference-2014',
-            'info_dict': {
-                'id': '1_a52wc67y',
-                'ext': 'flv',
-                'upload_date': '20150127',
-                'uploader_id': 'PremierMedia',
-                'timestamp': int,
-                'title': 'Os Guinness // Is It Fools Talk? // Unbelievable? Conference 2014',
-            },
-        },
         # Kaltura embed with single quotes
         {
             'url': 'http://fod.infobase.com/p_ViewPlaylist.aspx?AssignmentID=NUN8ZY',
             'info_dict': {
                 'id': '0_izeg5utt',
-                'ext': 'mp4',
+                'ext': 'f4v',
                 'title': '35871',
                 'timestamp': 1355743100,
                 'upload_date': '20121217',
                 'uploader_id': 'cplapp@learn360.com',
+                'view_count': int,
+                'duration': 3403,
+                'thumbnail': 'http://cfvod.kaltura.com/p/1067292/sp/106729200/thumbnail/entry_id/0_izeg5utt/version/100001',
```

Comment:
Is this static? If not, use regex
```suggestion
                'thumbnail': r're:https?://\w+.kaltura.com/',
```